src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandler.java

```diff
@@ -46,9 +45,14 @@ public void handle() {
                                 container,
                                 duplicate.fileName
                             );
+                            var id = envelopeService.createNewEnvelope(
+                                duplicate.container,
+                                duplicate.fileName,
+                                duplicate.blobCreatedAt
+                            );
 
+                            envelopeService.markAsRejected(id, "Duplicate");
```

Comment:
any ideas for rejection message?

Reply:
perhaps can inclue reference to original if possible?

Reply:
Duplicate envelope with zip filename {} for jurisdiction {} ?

Reply:
> Duplicate envelope with zip filename {} for jurisdiction {} ?

event has a foreign key to envelope which has filename and jurisdiction so this is not adding any extra info.

Reply:
So just Duplicate envelope then ?

Reply:
done. Extracted to constant and updated tests